css/quiz.css

```diff
@@ -0,0 +1,84 @@
+* {
```

Comment:
Nope you do want both.  Tab width just refers to the amount it indents when you _hit_ tab, indent using spaces then makes it so that it uses spaces to do the actual tab-initiated indentation.